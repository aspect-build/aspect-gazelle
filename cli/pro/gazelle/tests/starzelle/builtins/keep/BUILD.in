load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")

copy_to_bin(
    name = "ctb",
    srcs = [
        "kept.txt",  # keep
    ],
)

sh_library(
    name = "shells",
    srcs = [
        "kept.sh", # keep
    ],
)

sh_binary(
    name = "shells-bin",
    srcs = [
        "kept.sh", # keep
    ],
)

filegroup(
    name = "all-files",
    srcs = [
        "kept.sh", # keep
    ]
)
