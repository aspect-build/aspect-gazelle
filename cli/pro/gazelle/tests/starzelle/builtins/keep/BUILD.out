load("@aspect_bazel_lib//lib:copy_to_bin.bzl", "copy_to_bin")

copy_to_bin(
    name = "keep",
    srcs = [
        "a.txt",
        "kept.txt",  # keep
    ],
)
